lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -185,6 +185,16 @@ protected function doLoadMetadata($class, $parent, $rootEntityFound, array $nonS
                     $this->addNestedEmbeddedClasses($embeddableMetadata, $class, $property);
                 }
 
+                if ($embeddableClass['attributeOverrides']) {
```

Comment:
Which types can store `$embeddableClass['attributeOverrides']`? If it can be only `array`, the `if` isn't required.